### PR TITLE
[homekit] Augment README with some debugging help, links, and other help

### PR DESCRIPTION
**Pull request by timcharper**

### CODE REVIEW

File: addons/io/org.openhab.io.homekit/README.md [hunk 1]
```diff
@@ -87,19 +87,33 @@ Number DownstairsThermostatTargetTemperature "Downstairs Thermostat Target Tempe
 String DownstairsThermostatHeatingCoolingMode "Downstairs Thermostat Heating/Cooling Mode" (gDownstairsThermostat) [ "homekit:HeatingCoolingMode" ]
 ```
 
+## Common Problems
+
+### OpenHab2 Homekit hub shows up when I manually scan for devices, but Home app repots "can’t connect to device"
+
+If you see this error in the Home app, and don't see any log messages, it could be because your IP address is networkInterface is misconfigured.
```
[the remark this answers is not in the document]
Comment (wborn) [reply addons/io/org.openhab.io.homekit/README.md]: `networkInterface` is setting. See the [settings overview](https://github.com/openhab/openhab2-addons/blob/master/addons/io/org.openhab.io.homekit/README.md#overview-of-all-settings).

**Comment by wborn** [comment]
I've fixed the review comments @cweitkamp!

**Comment by timcharper** [comment]
Hi, I'm sorry, I just saw the request to sign off. I'll figure this out and do it.

**Comment by wborn** [comment]
It's also OK if you state that you agree with the DCO and provide us the `Signed-off-by:` line here in a comment @timcharper . Then we can add it while squashing/merging the commits. :smile:

File: addons/io/org.openhab.io.homekit/README.md [hunk 2]
```diff
@@ -9,14 +9,14 @@ Some accessory types require a specific set of characteristics.
...
-Your first step will be to create the homekit.cfg in your conf/services folder.
+Your first step will be to create the homekit.cfg in your `$OPENHAB_CONF/services` folder.
 At the very least, you will need to define a pin number for the bridge.
 This will be used in iOS when pairing. The pin code is in the form "###-##-###".
 Requirements beyond this are not clear, and Apple enforces limitations on eligible pins within iOS.
 At the very least, you cannot use repeating (111-11-111) or sequential (123-45-678) pin codes.
 If your home network is secure, a good starting point is the pin code used in most sample applications: 031-45-154.
 
-Other settings, such as using Fahrenheit temperatures, customizing the thermostat heat/cool/auto modes, and specifying the interface to advertise the Homekit bridge (which can be edited in PaperUI standard mode) are also illustrated in the following sample:
+Other settings, such as using Fahrenheit temperatures, customizing the thermostat heat/cool/auto modes, and specifying the interface to advertise the HomeKit bridge (which can be edited in PaperUI standard mode) are also illustrated in the following sample:
```
Comment (cweitkamp) [inline addons/io/org.openhab.io.homekit/README.md]: -> Paper UI

File: addons/io/org.openhab.io.homekit/README.md [hunk 3]
```diff
@@ -90,19 +90,19 @@ String DownstairsThermostatHeatingCoolingMode "Downstairs Thermostat Heating/Coo
 
 ## Common Problems
 
-### OpenHab2 Homekit hub shows up when I manually scan for devices, but Home app repots "can’t connect to device"
+**openHAB HomeKit hub shows up when I manually scan for devices, but Home app reports "can't connect to device"**
 
-If you see this error in the Home app, and don't see any log messages, it could be because your IP address is networkInterface is misconfigured.
-The OpenHab2 Homekit hub is advertised via mDNS.
-If you register an IP address that isn't reachable from your phone (such as `localhost`, `0.0.0.0`, `127.0.0.1`, etc.), then Home will be unable to reach OpenHab2.
+If you see this error in the Home app, and don't see any log messages, it could be because your IP address in the `networkInterface` setting is misconfigured.
```
Comment (cweitkamp) [inline addons/io/org.openhab.io.homekit/README.md]: -> settings

File: addons/io/org.openhab.io.homekit/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (cweitkamp) [reply addons/io/org.openhab.io.homekit/README.md]: Thanks for the hint.

File: addons/io/org.openhab.io.homekit/README.md (hunk 2 above)
Comment (wborn) [reply addons/io/org.openhab.io.homekit/README.md]: Fixed 2x!

**Comment by timcharper** [comment]
Signed-off-by: Tim Harper <timcharper@gmail.com> (github: timcharper)

**Pull request merged**